backend/ee/danswer/background/celery/celery_app.py

```diff
@@ -11,12 +11,25 @@
 from danswer.utils.logger import setup_logger
```

Comment:
Everything in this module is just refactoring the group and doc permission syncing to be seperate